raiden-ts/src/epics.ts

```diff
@@ -172,6 +172,11 @@ export function getLatest$(
     pluck('payload', 'stale'),
     startWith(initialStale),
   );
+  const settleTimeout$ = action$.pipe(
+    filter(contractSettleTimeout.is),
+    pluck('payload'),
+    startWith(0),
```

Comment:
AFAIK this value is not used at init time, and before init completes it'll have an actual value, so I think this isn't a problem, but if you want some peace of mind that we won't divide by zero somewhere and crash the Universe into a singularity, I can put the default deployment `settleTimeout` here just in case

Reply:
:heavy_check_mark: